libs/forgot-password/feature/src/lib/forgot-password.component.scss

```diff
@@ -29,7 +29,8 @@ ion-content {
 
 .form-details {
     position: relative;
-    width: 411px;
+    max-width: 411px;
```

Comment:
Are you sure this works? Tested on all small devices? 411px seems large still

Reply:
Dynamically sizes with padding on either side until it reaches 411px, and then the width of the field stays at 411px if the screen is larger